cylc/flow/etc/syntax/cylc.nanorc

```diff
@@ -0,0 +1,30 @@
+# How to use this file:
+# cylc get-resources cylc.nanorc ~/.config/nano/cylc.nanorc
+# Add the following to ~/.nanorc
+# include ~/.config/nano/cylc.nanorc
+
+# Supports `.cylc` files
+syntax "Cylc" "\.cylc$"
+
+## Multiline
+color yellow start="\"\"\"" end="\"\"\""
+color yellow start="'''" end="'''"
+color yellow start="\[" end="\]"
+
+## Values
+color yellow "=(.*)$"
+color green "=[^>]"
+color brightmagenta "=>|&|\|\\"
+
+## Valid headings
+color green "^\[.*\]$"
+color green "^\s{4}\[\[.*\]\]+$"
+color green "^\s{8}\[\[\[.*\]\]\]+$"
+
+## Comments (keep at the end of this file!)
+color cyan "#.*$"
+
+## Jinja2
+icolor red "^#!Jinja2"
```

Comment:
What is `icolor` as opposed to `color`?

Also, not sure red is a good choice as it tends to mean error/problem

Reply:
`icolor` is case insensitive regex so it will highlight `#!Jinja2` or `#!jinja2` or even `#!jInJa2` (which is valid!).

The choice of colors is somewhat limited on the older version of Nano we have at the MetOffice, although newer versions of nano offer more choice - when I try I get:

```
Valid colors are "green", "red", "blue",
"white", "yellow", "cyan", "magenta" and
"black", with the optional prefix "bright"
for foreground colors.

```

(see also https://www.nano-editor.org/dist/latest/nanorc.5.html)